examples/ExampleOne.js

```diff
@@ -4,74 +4,82 @@ import { View, Text } from 'react-native';
 import { ProgressSteps, ProgressStep } from 'react-native-progress-steps';
 
 class ExampleOne extends Component {
-  static navigationOptions = {
-    header: null
-  };
+	static navigationOptions = {
+		header: null
+	};
 
-  onNextStep = () => {
-    console.log('called next step');
-  };
+	defaultScrollViewProps = {
+		keyboardShouldPersistTaps: 'handled',
+		contentContainerStyle: {
+			flex: 1,
+			justifyContent: 'center'
+		}
+	}
 
-  onPaymentStepComplete = () => {
-    alert('Payment step completed!');
-  };
+	onNextStep = () => {
+		console.log('called next step');
+	};
 
-  onPrevStep = () => {
-    console.log('called previous step');
-  };
+	onPaymentStepComplete = () => {
+		alert('Payment step completed!');
+	};
 
-  onSubmitSteps = () => {
-    console.log('called on submit step.');
-  };
+	onPrevStep = () => {
+		console.log('called previous step');
+	};
 
-  render() {
-    return (
-      <View style={{ flex: 1, marginTop: 50 }}>
-        <ProgressSteps>
-          <ProgressStep
-            label="Payment"
-            onNext={this.onPaymentStepComplete}
-            onPrevious={this.onPrevStep}
-            centerContainer
-          >
-            <View style={{ alignItems: 'center' }}>
-              <Text>Payment step content</Text>
-            </View>
-          </ProgressStep>
-          <ProgressStep
-            label="Shipping Address"
-            onNext={this.onNextStep}
-            onPrevious={this.onPrevStep}
-            centerContainer
-          >
-            <View style={{ alignItems: 'center' }}>
-              <Text>Shipping address step content</Text>
-            </View>
-          </ProgressStep>
-          <ProgressStep
-            label="Billing Address"
-            onNext={this.onNextStep}
-            onPrevious={this.onPrevStep}
-            centerContainer
-          >
-            <View style={{ alignItems: 'center' }}>
-              <Text>Billing address step content</Text>
-            </View>
-          </ProgressStep>
-          <ProgressStep
-            label="Confirm Order"
-            onPrevious={this.onPrevStep}
-            onSubmit={this.onSubmitSteps}
-            centerContainer
-          >
-            <View style={{ alignItems: 'center' }}>
-              <Text>Confirm order step content</Text>
-            </View>
-          </ProgressStep>
-        </ProgressSteps>
-      </View>
-    );
-  }
+	onSubmitSteps = () => {
+		console.log('called on submit step.');
+	};
+
+	render() {
+		return (
+			<View style={{ flex: 1, marginTop: 50 }}>
+				<ProgressSteps>
+					<ProgressStep
+						label="Payment"
+						onNext={this.onPaymentStepComplete}
+						onPrevious={this.onPrevStep}
+						scrollViewProps={this.defaultScrollViewProps}
```

Comment:
```suggestion
						scrollViewProps={defaultScrollViewProps}
```
The "this" keyword needs to be removed for this to work properly.